modules/iap/versions.tf

```diff
@@ -23,5 +23,8 @@ terraform {
     kubernetes = {
       source  = "hashicorp/kubernetes"
     }
+    kubectl = {
```

Comment:
As mentioned offline, discuss further if we want to bring back the use of "alekc/kubectl"

Reply:
@blackzlq why do we need this? we worked hard to move away from this, let's not bring it back.

Reply:
this is an unofficial module that we probably shouldnt take a dependency on